src/features/tokens/components/top-relayer-tokens.js

```diff
@@ -0,0 +1,89 @@
+import { filter, flow, map, take } from 'lodash/fp';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+
+import { getDisplayCurrency } from '../../currencies/selectors';
+import AsyncTopTokensChart from './async-top-tokens-chart';
+import AutoReload from '../../../util/auto-reload';
+import getTokensWithStats from '../selectors/get-tokens-with-stats';
+import LoadingIndicator from '../../../components/loading-indicator';
+import tokensPropTypes from '../prop-types';
+
+// TODO: Deprecate this in favor of a markets table on relayer pages
```

Comment:
This is just a temporary component until a more robust relayer markets feature can be built.